components/html-block/html-block.js

```diff
@@ -178,16 +178,21 @@ class HtmlBlock extends LitElement {
 		super.connectedCallback();
 		if (this._contextObserverController) this._contextObserverController.hostConnected();
 
-		if (!this._templateObserver || this.noDeferredRendering) return;
+		if (!this._contentObserver || this.noDeferredRendering) return;
 
 		const template = this._findSlottedElement('TEMPLATE');
-		if (template) this._templateObserver.observe(template.content, { attributes: true, childList: true, subtree: true });
+		if (template) {
+			this._contentObserver.observe(template.content, { attributes: true, childList: true, subtree: true });
+		} else {
+			const container = this._findSlottedElement('DIV');
+			if (container) this._contentObserver.observe(container, { attributes: true, childList: true, subtree: true });
```

Comment:
This is technically different behaviour, since this will trigger the observer if a consumer adds an attribute onto the `div` itself. I suspect this case isn't worth worrying about, compared to the relative icky-ness of needing to attach the observer to every child element of the div just to ensure it and its children can use different options. 🤷